src/Desktop.Analyzers/Core/DesktopAnalyzersResources.resx

```diff
@@ -363,4 +363,52 @@
+    <comment>CA3075</comment>
+  </data>
+  <data name="DoNotUseDtdProcessingOverloadsMessage" xml:space="preserve">
+    <value>{0} uses unsafe overload of '{1}' method</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="DoNotUseDtdProcessingOverloads" xml:space="preserve">
+    <value>Insecure DTD Processing</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="DoNotUseSetInnerXmlMessage" xml:space="preserve">
+    <value>{0} uses the unsafe setter of InnerXml property of System.Xml.XmlDocument.</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="InsecureXmlDtdProcessing" xml:space="preserve">
+    <value>Insecure DTD processing in XML</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="ReviewDtdProcessingPropertiesMessage" xml:space="preserve">
+    <value>Property in {0} might be set from an untrusted source.</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="XmlDocumentWithNoSecureResolverMessage" xml:space="preserve">
+    <value>In {0} an XmlDocument instance is created without setting its XmlResolver property to a secure value.</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="XmlReaderCreateInsecureConstructedMessage" xml:space="preserve">
+    <value>An insecure XmlReaderSettings instance is provided to XmlReader.Create method.</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="XmlReaderCreateInsecureInputMessage" xml:space="preserve">
+    <value>In {0} a potentially insecure XmlReaderSettings instance is provided to XmlReader.Create method.</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="XmlReaderCreateWrongOverloadMessage" xml:space="preserve">
+    <value>{0} uses an insecure overload of XmlReader.Create which does not accept an XmlReaderSettings argument. </value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="XmlTextReaderConstructedWithNoSecureResolutionMessage" xml:space="preserve">
+    <value>{0} creates XmlTextReader instance with insecure default settings.</value>
+    <comment>CA3075</comment>
+  </data>
+  <data name="XmlTextReaderSetInsecureResolutionMessage" xml:space="preserve">
+    <value>XmlTextReader instance in {0} is set with insecure values.</value>
```

Comment:
I brought this up with @qinxgit offline, providing enclosing method name in a warning message is a FxCop convention, as it is a assembly analyzer, users usually needs those extra info to locate the issues in source code. Roslyn provides detailed warning location in source code, so this is no longer necessary, and removing those will make the message focus more on the issue with less noise. Plus it will simplify the code as well :)

I wouldn't make this a blocking comment for merge, but I do believe this will provide better experience to customers and benificial in longer term (as @qinxgit already noticed, getting enclosing method name requires some additional work when implementing those analyzer using `IOperation`)